modules/sbt-coursier/src/sbt-test/shared-1/credentials-sbt/build.sbt

```diff
@@ -10,3 +10,5 @@ credentials += Credentials(
 )
 
 libraryDependencies += "com.abc" % "test" % "0.1"
+
+resolvers += Resolver.file("space-repo", file(raw"/tmp/space the final frontier/repo"))(Resolver.ivyStylePatterns)
```

Comment:
Let's rename the scripted test to (something along the lines of) `credentials-sbt-and-space-char-resolver`, not to forget what it's about? Or put that in a new distinct scripted test?

Reply:
sure. I can make a new scripted test.